gui/elements/_window.py

```diff
@@ -36,10 +41,21 @@ def add_viewer(self):
         self._qt_central_widget.layout().addWidget(viewer.layers._qt)
         return viewer
 
-    def resize(self, *args):
+    def resize(self, width, height):
+        """Resizes the window.
+
+        Parameters
+        ----------
+        width : int
+            Width in pixels.
+        height : int
+            Height in pixels.
```

Comment:
How does Qt handle HiDPI displays?

Reply:
Ok thanks, my reading of that is, "don't worry about it, and mostly Qt will do the right thing.". =)

But, from my reading, I think this should be "height in *logical* pixels" (as opposed to *display* pixels.)